guides/release/code-editors/index.md

```diff
@@ -0,0 +1,61 @@
+During development, add Ember.js specifc extensions to your code editor for added functionality. Below is a list of some of the extensions available:  
+
+## Visual Studio Code
+
+Visual Studio Code is a code editor optimized for building and debugging modern web applications. VSC is one of the most popular text editors among ember developers.
+
+[Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable) - This provides autocompletions in templates and allows go-to-definition behavior within ember projects where tsserver wouldn't be able to help.
+
+[Glimmer Templates Syntax](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-glimmer-syntax) - Syntax formatting for glimmer templates.
+
+[Related Files Hopper](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) - This extension allows movement within a collection of related files with a single keystroke making development faster and efficient. This is especially useful for finding related files within embers components. 
+
+[EditorConfig for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) - Attempts to override user/workspace settings with settings found in .editorconfig files. The .editorconfig file helps developers define and maintain consistent coding styles between different editors and IDEs.
+
+[Prettier for Handlebars](https://marketplace.visualstudio.com/items?itemName=Alonski.prettier-for-handlebars-vscode) - Format your handlebars files with Prettier ...that's it! Note, because this uses the currently unreleased Prettier, it may break with future changes to Prettier.
+
+## vim
+
+Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X. 
+
+[ember.vim](https://github.com/dsawardekar/ember.vim) - Ember.vim is a Vim plugin for the Ember.js frontend framework inspired by vim-rails.
```

Comment:
```suggestion
[ember.vim](https://github.com/dsawardekar/ember.vim) -  a Vim plugin for the Ember.js frontend framework inspired by `vim-rails`.
```
Things in backticks don't get spellchecked. Or, we can add this to the `.dic` file in the Guides if we don't want to do backticks here.